creator/Documents/ScoreboardIntroduction.md

```diff
@@ -149,46 +149,45 @@ To get a list of all of the players in a world (including any fake ones you crea
 Use this command with the name of a player to get a list of that player's tracked objectives.
 
 ```
-/scoreboard players list <player>
+/scoreboard players list [playername: target]
 ``` 
 
 ### operation
 
 You can perform mathematical operations to calculate and assign scores.
 
 ```
-/scoreboard players operation Player1 objective operation Player2 objective
+/scoreboard players operation <TargetPlayer> <TargetObjective> <operation> <SourcePlayer> <SourceObjective>
```

Comment:
Here is the exception to the comment noted on line 85. The syntax for this command would be really long if I wrote it as presented in the game, not to mention it's not self-consistent.
```
/scoreboard players operation <targetName: target> <targetObjective: string> <operation: operator> <selector: target> <objective: string>
```
At the very least, the arguments need to have angled brackets around them.